pkgs/by-name/rs/rset/package.nix

```diff
@@ -1,37 +1,39 @@
 {
   lib,
   stdenv,
-  fetchurl,
+  fetchFromGitHub,
   coreutils,
   openssh,
   gnutar,
 }:
 
 stdenv.mkDerivation rec {
   pname = "rset";
-  version = "2.1";
+  version = "3.2";
 
-  src = fetchurl {
-    url = "https://scriptedconfiguration.org/code/${pname}-${version}.tar.gz";
-    sha256 = "0916f96afl8kcn2hpj4qhg92g2j93ycp2sb94nsz3q44sqc6ddhb";
+  src = fetchFromGitHub {
+    owner = "eradman";
+    repo = "rset";
+    tag = version;
+    hash = "sha256-b797R79aMopiPApTJ4Q3SP2MRjqCcNNO9BIxtuiNZks=";
   };
 
   patches = [ ./paths.patch ];
 
   postPatch = ''
     substituteInPlace rset.c \
-      --replace @ssh@       ${openssh}/bin/ssh \
-      --replace @miniquark@ $out/bin/miniquark \
-      --replace @rinstall@  $out/bin/rinstall \
-      --replace @rsub@      $out/bin/rsub
+      --replace-fail @ssh@       ${openssh}/bin/ssh \
+      --replace-fail @miniquark@ $out/bin/miniquark \
+      --replace-fail @rinstall@  $out/bin/rinstall \
+      --replace-fail @rsub@      $out/bin/rsub
 
     substituteInPlace execute.c \
-      --replace @ssh@     ${openssh}/bin/ssh \
-      --replace @ssh-add@ ${openssh}/bin/ssh-add \
-      --replace @tar@     ${gnutar}/bin/tar
+      --replace-fail @ssh@     ${openssh}/bin/ssh \
```

Comment:
Not wanting a change here but I am curious because I want to learn. What is the difference between replace and replace-fail?

Reply:
What is the difference between replace-warn and replace-fail?